client/samples/lib/sample/vcenter/vm/hardware/adapter/sata.rb

```diff
@@ -0,0 +1,93 @@
+# Copyright 2014-2017 VMware, Inc.  All Rights Reserved.
+# SPDX-License-Identifier: MIT
+require 'sample/framework'
+require 'vapi'
+require 'com/vmware/vcenter/vapi'
+require 'sample/vcenter/helper/vm_helper'
+
+class SATAConfiguration < SampleBase
+  HARDWARE_ADAPTER_MODULE = Com::Vmware::Vcenter::Vm::Hardware::Adapter
+  HARDWARE_ADAPTER_SATA_CLASS = HARDWARE_ADAPTER_MODULE::Sata
+  TITLE = 'SATA Configurations of a VM'
+  DESCRIPTION = <<EOL
+  Demonstrates how to configure virtual SATA adapters of a virtual machine.
+
+  Sample Prerequisites:
+  The sample needs an existing VM.
+EOL
+
+  # sample attributes
+  attr_reader :service_manager,:vm_name,:vm_id,:sata_svc,:satas_to_delete
+
+  def initialize
+    super(TITLE, DESCRIPTION, true)
+  end
+
+  def inject_options
+    option_parser.on("-v", "--vm-name NAME","Name of the VM") do |value|
+      options[:vm_name] =  value
+      end 
```

Comment:
This should be indented the same level as line 27 (`option_parser.on("-v", ...`).